### ISSUE TITLE
Upgrading Google-cloud version

### ISSUE THREAD

**Issue by MMMarcy** [GoogleCloudPlatform/pubsub#75]
The current google-cloud library version (0.9.3-alpha) needs to be upgraded.

**Comment by kir-titievsky** [comment]
@urshunkeler Any sense of how to proceed with getting the tests to work?

**Comment by kir-titievsky** [comment]
Blocked on #56

**Comment by MMMarcy** [comment]
Pull request merged into jsm_publisher_test_suite. Will close this issue for now and update the project status accordingly

**Comment by urshunkeler** [comment]
Upgrading the google-cloud version without addressing the question of where to take the pub/sub emulator from does not really solve the issue as this will simply disable most tests. In the document I submitted for comments and which should be discussed in the next GVC meeting I propose to fork the pub/sub emulator and put it in our test directory. We can then probably remove the dependency to the google-cloud-java project altogether. Reopening for now as I am undoing the code changes to work on issue #87.
